composer.json

```diff
@@ -21,7 +21,8 @@
     "ext-openssl": "*",
     "google/apiclient": "^2.9.1",
     "google/apiclient-services": "^0.180",
-    "guzzlehttp/guzzle": "^5.3.3"
+    "guzzlehttp/guzzle": "^5.3.3",
+    "true/punycode": "^2.0 <2.1.1"
```

Comment:
This looks a bit odd because v2.1.1 added `symfony/polyfill-mbstring` as a dependency. This is unnecessary because WP polyfills multibyte string functions internally but also, added unnecessary complexity to the implementation because of an autoloaded functions file the package includes. The polyfill functions wouldn't work anyways because of our namespace prefixing so it was simpler to just avoid this change altogether and use the previous version. This was the only change in 2.1.1.